src/background/services/paymentSession.ts

```diff
@@ -172,6 +183,15 @@ export class PaymentSession {
             }
           })
 
+          if (this.intervalInMs > 1000) {
+            this.tabState.saveOverpaying(
```

Comment:
I'm not following how we define "overpaying". We can just store that state always? It's ok if `waitTime` is 0. That data can be stored bookkeeping only (in memory only).

Reply:
If we are paying every second, there is no use of storing all the data.  cc: @raducristianpopa